langchain4j-ollama/src/test/java/dev/langchain4j/model/ollama/OllamaClientIT.java

```diff
@@ -48,4 +48,28 @@ void should_respond_with_model_information() {
         assertThat(modelDetailsResponse.getDetails().getFormat()).isEqualTo("gguf");
         assertThat(modelDetailsResponse.getDetails().getFamily()).isEqualTo("llama");
     }
+
+    @Test
+    void should_delete_model() {
```

Comment:
Did you check if this test slows down all Ollama ITs?

Reply:
I ran all Ollama tests and did not see any slowdowns. All tests for Ollama run on my computer in about 1 minute. 50 sec is just for OllamaChatModelVisionIT